<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>768</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QFrame" name="frame_mode">
      <property name="minimumSize">
       <size>
        <width>750</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>750</width>
        <height>90</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_decodeMode">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>111111</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
background-color: #84f542;
color: black;</string>
         </property>
         <property name="text">
          <string>  Decode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_encodeMode">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>11111</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
background-color: #b4becf;
color: black;</string>
         </property>
         <property name="text">
          <string>  Encode</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_main">
      <property name="minimumSize">
       <size>
        <width>750</width>
        <height>410</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>750</width>
        <height>410</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextEdit" name="textEdit_input">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>160</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>160</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Type (or paste) here...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="widget_swapButtons">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_decode">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="btn_decode">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #155708;
color: white;</string>
             </property>
             <property name="text">
              <string>&lt; DECODE &gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_encode">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="btn_encode">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #155708;
color: white;</string>
             </property>
             <property name="text">
              <string>&lt; ENCODE &gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_output">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>160</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>160</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Result goes here...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
